packages/api/src/clickhouse/index.ts

```diff
@@ -1017,6 +1024,41 @@ export const buildMetricSeriesQuery = async ({
     ],
   );
 
+  const quantile =
+    aggFn === AggFn.P50
+      ? '0.5'
+      : aggFn === AggFn.P90
+      ? '0.90'
+      : aggFn === AggFn.P95
+      ? '0.95'
+      : '0.99';
+
+  const histogramMetricSource = SqlString.format(
```

Comment:
we can also add a comment here to let people know the implementation doesnt linear interpolate the bucket values